remote-api/src/main/java/org/alfresco/rest/api/nodes/NodeVersionRenditionsRelation.java

```diff
@@ -117,6 +134,30 @@ public BinaryResource readProperty(String nodeId, String versionId, Parameters p
         return renditions.getContent(nodeRef, versionId, renditionId, parameters);
     }
 
+    @Operation ("request-direct-access-url")
+    @WebApiParam (name = "directAccessUrlRequest", title = "Request direct access url", description = "Options for direct access url request", kind = ResourceParameter.KIND.HTTP_BODY_OBJECT)
+    @WebApiDescription(title = "Request content url",
+            description="Generates a direct access URL.",
+            successStatus = HttpServletResponse.SC_OK)
+    public DirectAccessUrl requestContentDirectUrl(String nodeId, String versionId, DirectAccessUrlRequest directAccessUrlRequest, Parameters parameters, WithResponse withResponse)
```

Comment:
It would be more readable if parameters were in separate lines.

Reply:
done